app/client/components/payment/update/updatePaymentFlow.tsx

```diff
@@ -128,32 +129,71 @@ const PaymentMethodRadioButton = (props: PaymentMethodRadioButtonProps) => {
     }
   `;
 
-  return (
+  const label = () => (
     <div
       css={css`
         display: flex;
+        align-items: center;
+      `}
+    >
+      <span
+        css={css`
+          flex: 1;
+        `}
+      >
+        {props.paymentMethod}
+      </span>
+      <div
+        css={css`
+          display: none;
+          ${minWidth.mobileMedium} {
+            display: block;
+            margin: auto;
+          }
+        `}
+      >
+        <div
+          css={css`
+            display: flex;
+          `}
+        >
+          {getLogos(props.paymentMethod)}
+        </div>
+      </div>
+    </div>
+  );
+
+  return (
+    <div
+      css={css`
         border-radius: 4px;
-        padding: ${space[4]}px;
+
         margin-bottom: ${space[4]}px;
         ${isChecked ? radioIsChecked : radioDefault}
 
         label {
           min-height: 0;
-          flex: 1;
+          padding: ${space[4]}px;
 
           div {
             font-weight: bold;
           }
         }
 
+        :hover {
+          -webkit-box-shadow: inset 0px 0px 0px 4px ${brand[500]};
+          -moz-box-shadow: inset 0px 0px 0px 4px ${brand[500]};
```

Comment:
I don't think you need to worry about prefixing `box-shadow` anymore as it's only needed for _really_ old browsers. (Doesn't Emotion include auto-prefixing anyway?)

Reply:
good point. this was something i wanted to ask Richard about. I'll google it.

Reply:
yep, seems like emotion includes auto-prefixing by default